python/foglamp/tasks/north/sending_process.py

```diff
@@ -226,9 +226,9 @@ class SendingProcess(FoglampProcess):
             "order": "7"
         },
         "blockSize": {
-            "description": "Bytes to send in each transmission",
+            "description": "Number of readings to send in each transmission",
             "type": "integer",
-            "default": "500",
+            "default": "5000",
```

Comment:
Yes, this is good. Even 10K is best suited, with compression. it should always be (configured)  so that payloadSize < 192KB. The max payload size we have in single row is for systemInfo and 5000 was perfect with millions of readings.